lib/Parse/ParsedSyntaxNodes.cpp.gyb

```diff
@@ -27,15 +27,15 @@ using namespace swift::syntax;
 %   for child in node.children:
 %     if child.is_optional:
 Optional<Parsed${child.type_name}>
-Parsed${node.name}::getDeferred${child.name}() {
-  auto &RawChild = getRaw().getDeferredChildren()[${node.name}::Cursor::${child.name}];
+Parsed${node.name}::getDeferred${child.name}(const SyntaxParsingContext *SyntaxContext) {
+  auto RawChild = getRaw().getDeferredChild(${node.name}::Cursor::${child.name}, SyntaxContext);
   if (RawChild.isNull())
     return None;
   return Parsed${child.type_name} {RawChild.copyDeferred()};
```

Comment:
ditto

Reply:
If you remove them, `copyDeferred()` is not used anymore...

Reply:
See above.